### ISSUE TITLE
[BUG] Installing salt on Amazon Linux 2 - Python dependency issue

### ISSUE THREAD

**Issue by winternet-studio** [saltstack/salt#61162]
**Description**
I'm following the installation instructions on https://repo.saltproject.io/#amzn for installing salt-minion on Amazon Linux 2. But I get error messages about Python.

**Setup**
On AWS I created an EC2 instance, using the AMI `amzn2-ami-hvm-2.0.20211005.0-arm64-gp2`.

**Steps to Reproduce the behavior**

`sudo rpm --import https://repo.saltproject.io/py3/amazon/2/x86_64/latest/SALTSTACK-GPG-KEY.pub`
`curl -fsSL https://repo.saltproject.io/py3/amazon/2/x86_64/latest.repo | sudo tee /etc/yum.repos.d/salt-amzn.repo`
`sudo yum clean expire-cache`

As I then run `yum install salt-minion` I get the following dependency errors:

```
Error: Package: salt-3004-1.amzn2.noarch (salt-latest-repo)
           Requires: python3-pyyaml
Error: Package: salt-3004-1.amzn2.noarch (salt-latest-repo)
           Requires: python37-pycryptodomex
Error: Package: salt-3004-1.amzn2.noarch (salt-latest-repo)
           Requires: python3-zmq >= 17.0.0
Error: Package: python3-contextvars-2.4-1.amzn2.noarch (salt-latest-repo)
           Requires: python3-immutables
Error: Package: salt-3004-1.amzn2.noarch (salt-latest-repo)
           Requires: python3-msgpack >= 0.4
```

If I try to manually install eg. `python3-pyyaml` it says package is not found. Have even tried enabling EPEL packages.

Btw, is it an issue that the AMI is using arm?

**Comment by waynew** [comment]
@winternet-studio thanks for the report! I'm prepping for [saltconf](https://saltconf.com) right now (registration is free if you haven't yet!) for tomorrow/thursday, but I'll se about reproducing this probably early next week!

**Comment by bryceml** [comment]
arm64 is not currently supported for amazonlinux 2, but is on the roadmap.  Closing as a duplicate of https://github.com/saltstack/salt/issues/59779
